### PR TITLE
Observation new page

### PR DESCRIPTION
**Pull request by siba-tunerlabs**

### CODE REVIEW

File: src/components/selectable-list/selectable-list.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ * See https://angular.io/api/core/Component for more info on Angular
+ * Components.
+ */
+@Component({
+  selector: 'selectable-list',
+  templateUrl: 'selectable-list.html'
+})
+export class SelectableListComponent {
+  @Input() selectableList;
```
Comment (kiranharidas187) [inline src/components/selectable-list/selectable-list.ts:14]: Is this required?

File: src/components/selectable-list/selectable-list.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  text: string;
+
+  constructor() {
+    console.log('Hello SelectableListComponent Component');
+    this.text = 'Hello World';
+  }
+  doInfinite(infiniteScroll) {
+    console.log("doInfinite function called");
+    setTimeout(() => {
+      for (let i=0 ; ( i < 10 ) &&( this.itemsPerShot < this.list.length); i++ ) {
```
Comment (kiranharidas187) [inline src/components/selectable-list/selectable-list.ts]: recheck the logic

File: src/app/app.scss [hunk 3]
```diff
@@ -20,6 +20,62 @@ $dark-grey-color: #808080;
...
 $font-size-large: 18px;
 $white-color : #fff;
+// $step-circle-active-background-color: #a63936 ;
+
+
+
+
+$stepper-prefix: ionic-stepper;
+$step-prefix: ionic-step;
+
```
Comment (kiranharidas187) [inline src/app/app.scss]: remove this if not required

File: src/pages/observations/add-observation-form/add-observation-form.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  <header [title]="'headings.addObservation'|translate" [hideBack]="false" [showClose]="false" [disableNetwork]="true"
+    [showMenu]="false">
+  </header>
+</ion-header>
+
+<ion-content padding-top>
+  <ion-stepper [mode]="'horizontal'" #stepper (selectIndexChange)="selectChange($event)">
+
+    <ion-step label="Entity Type">
+      <ion-list radio-group [(ngModel)]="entityType">
```
Comment (kiranharidas187) [inline src/pages/observations/add-observation-form/add-observation-form.html:13]: Add e heading

File: src/pages/observations/add-observation-form/add-observation-form.ts [hunk 5]
```diff
@@ -0,0 +1,377 @@
...
+      this.listOfFrameWork = success.result;
+      console.log(JSON.stringify(this.listOfFrameWork))
+      solutionFlag = true;
+    }, error => {
+
+    });
+    return solutionFlag;
+  }
+
+  getCurrentLocation() {
```
Comment (kiranharidas187) [inline src/pages/observations/add-observation-form/add-observation-form.ts]: rename

File: src/pages/observations/add-observation-form/add-observation-form.ts [hunk 6]
```diff
@@ -0,0 +1,377 @@
...
+    }).catch(() => {
+      this.localStorage.setLocalStorage('draftObservation', [obsData]);
+
+    })
+  // }
+  this.app.getRootNav().pop();
+  }
+
+  creatPayLoad(){
+    let payLoad = this.addObservationForm.getRawValue();
```
Comment (kiranharidas187) [inline src/pages/observations/add-observation-form/add-observation-form.ts:322]: Modify on the basis of API DOc

File: src/pages/observations/add-observation-form/school-list/school-list.ts [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+ *
+ * See https://ionicframework.com/docs/components/#navigation for more info on
+ * Ionic pages and navigation.
+ */
+
+@Component({
+  selector: 'page-school-list',
+  templateUrl: 'school-list.html',
+})
+export class SchoolListPage {
```
Comment (kiranharidas187) [inline src/pages/observations/add-observation-form/school-list/school-list.ts]: rename to EntityListing page

File: src/pages/observations/draft-observation/draft-observation.ts [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
```
Comment (kiranharidas187) [inline src/pages/observations/draft-observation/draft-observation.ts]: remove if not required

File: src/pages/observations/my-observation/my-observation.html [hunk 9]
```diff
@@ -0,0 +1,109 @@
+<ion-header>
```
Comment (kiranharidas187) [inline src/pages/observations/my-observation/my-observation.html:1]: remove

File: src/pages/observations/my-observation/my-observation.module.ts [hunk 10]
```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
```
Comment (kiranharidas187) [inline src/pages/observations/my-observation/my-observation.module.ts:1]: remove if not required

File: src/components/selectable-list/selectable-list.ts (hunk 1 above)
Comment (siba-tunerlabs) [reply src/components/selectable-list/selectable-list.ts:14]: done

File: src/components/selectable-list/selectable-list.ts (hunk 2 above)
Comment (siba-tunerlabs) [reply src/components/selectable-list/selectable-list.ts]: done

File: src/app/app.scss (hunk 3 above)
Comment (siba-tunerlabs) [reply src/app/app.scss]: done

File: src/pages/observations/add-observation-form/add-observation-form.html (hunk 4 above)
Comment (siba-tunerlabs) [reply src/pages/observations/add-observation-form/add-observation-form.html:13]: done

File: src/pages/observations/add-observation-form/add-observation-form.ts (hunk 5 above)
Comment (siba-tunerlabs) [reply src/pages/observations/add-observation-form/add-observation-form.ts]: done

File: src/pages/observations/add-observation-form/add-observation-form.ts (hunk 6 above)
Comment (siba-tunerlabs) [reply src/pages/observations/add-observation-form/add-observation-form.ts:322]: done

File: src/pages/observations/add-observation-form/school-list/school-list.ts (hunk 7 above)
Comment (siba-tunerlabs) [reply src/pages/observations/add-observation-form/school-list/school-list.ts]: done

File: src/pages/observations/draft-observation/draft-observation.ts (hunk 8 above)
Comment (siba-tunerlabs) [reply src/pages/observations/draft-observation/draft-observation.ts]: done

**Pull request merged**